src/components/settings/settings.tsx

```diff
@@ -65,12 +69,149 @@ class Settings extends React.Component<Props, State> {
+
+  /**
+   * Method for transportation accordion menu item
+   */
+  private accordionMenuTransportation = () => {
+    return (
+      <Accordion defaultExpanded={ true }>
+        <AccordionSummary expandIcon={ <ExpandMoreIcon /> }>
+          <Typography>{ strings.settingsDrawer.transportation }</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Typography>{ strings.settingsDrawer.descriptionTransportation }</Typography>
+          <FormGroup>
+            { this.menuCheckbox(strings.walking) }
+            { this.menuCheckbox(strings.wheelerchair) }
+            { this.menuCheckbox(strings.walking) }
+            { this.menuCheckbox(strings.wheelerchair) }
+          </FormGroup>
+        </AccordionDetails>
+      </Accordion>
+    );
+  }
+
+  /**
+   * Method for medical accordion menu item
+   */
+  private accordionMenuMedical = () => {
+    return (
+      <Accordion defaultExpanded={ true }>
+        <AccordionSummary expandIcon={ <ExpandMoreIcon /> }>
+          <Typography>{ strings.settingsDrawer.medical }</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Typography>{ strings.settingsDrawer.descriptionMedical }</Typography>
+          <FormGroup>
+            { this.menuCheckbox(strings.medicalConditions.asthma) }
+            { this.menuCheckbox(strings.medicalConditions.ischaemicHeartDisease) }
+            { this.menuCheckbox(strings.medicalConditions.copd) }
+          </FormGroup>
+        </AccordionDetails>
+      </Accordion>
+    );
+  }
+
+  /**
+   * Method for rendering checkbox
+   * @param label checkbox label string
+   */
+  private menuCheckbox = (label: string) => {
+
+    return (
+      <FormControlLabel
+        control={ <Checkbox color="primary" /> }
+        label={ label }  
+        labelPlacement= "start"
+      />
+    );
+  }
+
+  /**
+   * Method for medical accordion menu item
+   */
+  private accordionMenuCustom = () => {
+    return (
+      <Accordion>
+        <AccordionSummary expandIcon={ <ExpandMoreIcon /> }>
+          <Typography>{ strings.settingsDrawer.custom }</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Typography>{ strings.settingsDrawer.descriptionCustom }</Typography>
+          { this.pollutantSlider(strings.pollutants.carbonMonoxide) }
+          { this.pollutantSlider(strings.pollutants.ozone) }
+          { this.pollutantSlider(strings.pollutants.sulfurOxide) }
+          { this.pollutantSlider(strings.pollutants.nitrogenOxide) }
+          { this.pollutantSlider(strings.pollutants.PM10) }
+          { this.pollutantSlider(strings.pollutants.PM25) }
+          <Button
+            color="primary"
+            variant="contained"
+            fullWidth={ true }
+          >
+            { strings.common.save }
+          </Button>
+        </AccordionDetails>
+      </Accordion>
+    );
+  }
+
+  /**
+   * Method for rendering pollutant slider
+   * @param label slider label string
```

Comment:
```suggestion
   * Method for rendering pollutant slider
   *
   * @param label slider label string
```

Good to separate description and params